app/services/npc_service.rb

```diff
@@ -1,30 +1,15 @@
 class NPCService
-  def self.get_npc_name
```

Comment:
This API call just got annoying because of the limits on the number of calls, and the results weren't all that interesting, so I took it out and we're using Faker now.